code/data-processing/get_imputed_data.R

```diff
@@ -0,0 +1,187 @@
+library(covidData)
+library(dplyr)
+library(here)
+
+setwd(here())
+
+# find  observations with negative inc
+get_negative_cases <- function(data) {
+  locations <- c()
+  dates <- c()
+  
+  for (i in 1:nrow(data)) {
+    if (data[i, ]$inc < 0) {
+      locations <- c(locations, as.character(data[i, ]$location))
+      dates <- c(dates, as.character(data[i, ]$date))
+    }
+  }
+  
+  adjustments <- data.frame(fips = locations, dates = as.Date(dates))
+  
+  return(adjustments)
+}
```

Comment:
I know we went back and forth about this, but it seems like we ended up adding this function to the package as an exported function.  If that's a decision we're good with, we can delete it from here.

```suggestion
```